### PR TITLE
Add buan.sql no readme

### PR DESCRIPTION
**Pull request by igotnowifi-official**

### CODE REVIEW

File: code/.DS_Store
Comment (ananyasingh7) [inline code/.DS_Store:1]: Don't push this file

File: code/db/BUAN.sql [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+-- Table: session_log (bridge)
+CREATE TABLE session_log (
+    sessionlog_id SERIAL PRIMARY KEY,
+    user_id INTEGER NOT NULL REFERENCES users(user_id),
+    session_id INTEGER NOT NULL REFERENCES sessions(session_id),
+    is_active BOOLEAN NOT NULL,
+    is_saved BOOLEAN NOT NULL
+);
+
+
```
Comment (ananyasingh7) [inline code/db/BUAN.sql:158]: Can you add the following as well?:

```
CREATE TABLE chat_history (
    chat_id SERIAL PRIMARY KEY,
    user_id INTEGER REFERENCES users(user_id),
    timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
    conversation JSONB NOT NULL,
    metadata JSONB
);

**Review by ananyasingh7 — CHANGES_REQUESTED** [review]

**Review by igotnowifi-official — COMMENTED** [review]
changes updated

**Comment by ananyasingh7** [comment]
lgtm

**Review by ananyasingh7 — APPROVED** [review]

**Pull request merged**